plugins/optimization-detective/detect.js

```diff
@@ -310,6 +310,23 @@ function extendElementData( xpath, properties ) {
 	Object.assign( elementData, properties );
 }
 
+/**
+ * Compresses a string using CompressionStream API.
+ *
+ * @param {string} string String to compress.
```

Comment:
Let's add the `-` for readability. Many of the other functions use this with the `@param` tags, although not all (but they should).

```suggestion
 * @param {string} string - String to compress.
```